contracts/src/core/EigenDAServiceManager.sol

```diff
@@ -121,58 +105,29 @@ contract EigenDAServiceManager is Initializable, OwnableUpgradeable, EigenDAServ
         }
 
         // store the metadata hash
-        uint96 fee = 0;
         uint32 batchIdMemory = batchId;
         bytes32 batchHeaderHash = batchHeader.hashBatchHeader();
-        batchIdToBatchMetadataHash[batchIdMemory] = EigenDAHasher.hashBatchHashedMetadata(batchHeaderHash, signatoryRecordHash, fee, uint32(block.number));
+        batchIdToBatchMetadataHash[batchIdMemory] = EigenDAHasher.hashBatchHashedMetadata(batchHeaderHash, signatoryRecordHash, uint32(block.number));
 
-        emit BatchConfirmed(reducedBatchHeaderHash, batchIdMemory, fee);
+        emit BatchConfirmed(reducedBatchHeaderHash, batchIdMemory);
```

Comment:
It seems a breaking change, what's the plan to handle this at indexer? I saw G commented this out as well but not yet addressed.

Reply:
Should be covered here: https://github.com/Layr-Labs/eigenda/pull/192